setup.py

```diff
@@ -62,6 +62,7 @@ def get_git_sha():
         'gunicorn==19.7.1',
         'idna==2.5',
         'markdown==2.6.8',
+        'pyopenssl==17.4.0',
```

Comment:
Not sure if we should tax the package with this even though some folks may not need this or even use SSL to run Superset (most likely the proxy/LB is taking care of that)